screen-orientation/ios/Plugin/ScreenOrientation.swift

```diff
@@ -84,12 +88,13 @@ public class ScreenOrientation: NSObject {
         switch orientationType {
         case "any":
             return UIInterfaceOrientationMask.all
-        case "landscape", "landscape-primary":
+        case "landscape":
+            return UIInterfaceOrientationMask.landscape
+        case "landscape-primary":
+            return UIInterfaceOrientationMask.landscapeLeft
```

Comment:
Must Fix: `landscape-primary` must be `UIInterfaceOrientationMask.landscapeRight`.
Please add the comment back.